server/test/services/question/types/QuestionDefinitionTest.java

```diff
@@ -166,11 +164,21 @@ public void isRepeated_true() {
 
   @Test
   public void newQuestionHasCorrectFields() throws Exception {
-    QuestionDefinition question = builder.setId(123L).build();
+    QuestionDefinition question =
+        new QuestionDefinitionBuilder()
```

Comment:
suggestion (non-blocking):  Since `QuestionDefinitionBuilder` is deprecated, it might be better to use `QuestionDefinitionConfig.Builder` to build this `QuestionDefinition`.

Reply:
Didn't realize it was deprecated, thanks! Fixed.